nodejs/week1/src/backend/routes/meal.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const app = express();
+
+const mealsJson = require("../data/meals.json"); //here you require the data from a JSON file
+const reviewsJson = require("../data/reviews.json"); //here you require the data from a JSON file
+
+// Here you map the meals and matches the meal id with the reviews mealId
+const mealsWithReviews = mealsJson.map(item => {
+    item.reviews = reviewsJson.filter(element => element.mealId === item.id)
+    return item;
+});
+
+//here you get a random meal
+const indexRandomMeal = Math.floor(Math.random()* mealsWithReviews.length);
```

Comment:
Isn't it the same random meal all the time? If it is, try to understand why and fix it 🙂

The same question applies to all the places, where you have random-item logic.